docs/migration.md

```diff
@@ -0,0 +1,134 @@
+# Migratie
+Op dit moment hebben de meeste repositories een eigen set aan coding standard tools als dependencies gedefinieerd in
+de `composer.json`. Ook bevatten deze repositories afzonderlijk van elkaar een configuratie met ruleset die gebruikt
+moet worden.
+
+De volgende stappen zorgen voor een soepele migratie naar het gebruiken van de PAQT coding standards.
+
+## Installeer `paqtcom/coding-standards` package
+De kans is groot dat bepaalde versies van de coding standard tools al geïnstalleerd zijn. Daarom gaan we eerst
+een normale installatie uitvoeren van de `paqtcom/coding-standards` package waarbij alleen de nog ontbrekende
+dependencies zullen worden geïnstalleerd. Alle benodigde dependencies die al zijn geïnstalleerd, en binnen de versie
+range van `paqtcom/coding-standards` package vallen, zullen niet aangepast worden.
+
+Het doel is om ervoor te zorgen dat we de minste afwijkingen in gedrag en resultaat van de tools krijgen, door zoveel
+mogelijk de beschikbare versies van dependencies te gebruiken.
+
+```shell
+composer require paqtcom/coding-standards --dev
+```
+
+## Verwijder overbodige directe dependencies
+De `paqtcom/coding-standards` package is nu verantwoordelijk voor de juiste dependencies die nodig zijn voor het
+gebruiken van de coding standard tools en dus kunnen de directe dependencies die nog aanwezig zijn in de `composer.json`
+verwijderd worden.
+
+```shell
+composer remove squizlabs/php_codesniffer friendsofphp/php-cs-fixer phpmd/phpmd phpstan/phpstan nunomaduro/larastan sebastian/phpcpd --dev --no-update
+```
+
+## Valideren van de tools
+Alle voorgaande aanpassingen zouden geen effect moeten hebben op het gedrag en het resultaat van de coding standard
+tools zoals die al gebruikt werden. Dit is dus een goed moment om te checken of alles nog werkt zoals het zou moeten.
+
+## Installeer `paqtcom/paqt-assistant` package
```

Comment:
je verwijst naar een private repo vanuit een public repo? Misschien naar interne documentatie verplaatsen voor nu?

Persoonlijk denk ik dat het wat vroeg is om het al in de documentatie neer te zetten zonder dit met andere leads door te nemen. Zelf gebruik in een composer plugin voor het bijhouden van de coding standaard bij verschillende projecten en zal ik het niet nodig hebben, maar ik weet vrij zeker dat niet elke lead dat belangrijk vind.

Reply:
> je verwijst naar een private repo vanuit een public repo? Misschien naar interne documentatie verplaatsen voor nu?

Goed punt inderdaad, zal er een Confluence pagina van maken.